src/app-logic/url-handling.js

```diff
@@ -176,12 +184,11 @@ export function urlStateToUrlObject(urlState: UrlState): UrlObject {
     query.timelineType = 'stack';
   }
 
-  const localTrackOrderByPid = '';
+  let localTrackOrderByPid = '';
```

Comment:
This started (correctly) throwing an error whenever I changed the type definition.